tests/integration/tests/poi.js

```diff
@@ -178,6 +165,50 @@ test('display details about the poi on a poi click', async () => {
   expect(wiki_block).not.toBeFalsy()
 })
 
+async function select_poi_1(page) {
+  await page.evaluate(() => {
+    window.MAP_MOCK.evented.prepare('click', 'poi-level-1',  {originalEvent : {clientX : 1000},features : [{properties :{global_id : 1}}]})
+  })
+  await page.click('#mock_poi')
+  await wait(300)
+}
+
+test('add a poi as favorite and find it back in the favorite menu', async () => {
+  expect.assertions(7)
+  await page.goto(APP_URL)
+
+  // we select a poi and 'star' it
+  await select_poi_1(page)
+  let poiPanel = await page.waitForSelector('.poi_panel__title')
+  expect(poiPanel).not.toBeFalsy()
+  await page.click('.poi_panel__actions__icon__store')
+
+  // we check that the first favorite item is our poi
+  await toggleFavoritePanel(page)
+  let fav = await getFavorites(page)
+  expect(fav).toHaveLength(1)
+  expect(fav[0].title).toEqual("Musée d'Orsay")
+  expect(fav[0].desc).toEqual("musée")
+  expect(fav[0].icons).toContainEqual("icon-museum")
+
+  // we then reopen the poi panel and 'unstar' the poi.
+  await wait(100)
```

Comment:
I don't understand why this wait is needded (the `toggleFavoritePanel` function wait `300` for the end of the animation) but If I remove it the test fail :grin: